src/Microsoft.Framework.Logging/Logger.cs

```diff
@@ -31,7 +31,14 @@ public void Log(LogLevel logLevel, int eventId, object state, Exception exceptio
             {
                 foreach (var logger in _loggers)
                 {
-                    logger.Log(logLevel, eventId, state, exception, formatter);
+                    try
+                    {
+                        logger.Log(logLevel, eventId, state, exception, formatter);
+                    }
+                    catch
+                    {
+                        // ignore exceptions thrown by a logger and continue calling other loggers
```

Comment:
Can we instead ensure that we log to all the loggers but still fail if any of them failed?